claims_hosp/HospClaims/automate/secrets.py

```diff
@@ -0,0 +1 @@
+../../secrets.py
```

Comment:
Does this need to get us back to `claims_hosp` or just to `HospClaims`?

Reply:
The goal is for the new Ansible to deploy the `secrets.py` file into the indicator root directory, so back to `claims_hosp`, I think.